### PR TITLE
Create B827EBFFFEA488FE.json

### PR DESCRIPTION
**Pull request by lostall2**

### CODE REVIEW

File: B827EBFFFEA488FE.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "server_address": "eu1.cloud.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 49.34,
+    "ref_longitude": 8.15,
+    "ref_altitude": 135,
+    "contact_email": "",
```
Comment (gonzalocasas) [inline B827EBFFFEA488FE.json:15]: please add your email here.

**Review by gonzalocasas — CHANGES_REQUESTED** [review]
email missing

**Review by lostall2 — COMMENTED** [review]
done

**Comment by gonzalocasas** [comment]
Thanks!

**Pull request merged**